src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -331,7 +193,7 @@ private[snowflake] class JDBCWrapper {
     })
   }
 
-  def executePreparedInterruptibly(conn: Connection, sql: String): Boolean = {
+  def executePreparedInterruptibly(conn: ServerConnection, sql: String): Boolean = {
```

Comment:
In production code, use `ServerConnection` instead of `Connection` and `SnowflakeConnectionV1`.